flow/layers/layer.h

```diff
@@ -61,6 +61,9 @@ struct PrerollContext {
   // These allow us to track properties like elevation, opacity, and the
   // prescence of a texture layer during Preroll.
   bool has_texture_layer = false;
+
+  int picture_count = 0;
+  size_t picture_bytes = 0;
```

Comment:
Please add doc comments.

However, I'm wondering if this will be all that helpful - if you have one pathologically large picture you'll want to know which one it is right? In a complex application/scene that may be non-trivial to discover. But if we try to trace per picture that can end up being a lot of events per frame...